src/main/resources/assets/copper_golem/lang/en_us.json

```diff
@@ -1,20 +1,20 @@
 {
-   "block.copper_golem.oxidized_copper_button": "Oxidized Copper Button",
-   "block.copper_golem.weathered_copper_button": "Weathered Copper Button",
-   "block.copper_golem.exposed_copper_button": "Exposed Copper Button",
-   "block.copper_golem.copper_button": "Copper Button",
+  "block.copper_golem.oxidized_copper_button": "Oxidized Copper Button",
+  "block.copper_golem.weathered_copper_button": "Weathered Copper Button",
+  "block.copper_golem.exposed_copper_button": "Exposed Copper Button",
+  "block.copper_golem.copper_button": "Copper Button",
 
-   "block.copper_golem.waxed_oxidized_copper_button": "Waxed Oxidized Copper Button",
-   "block.copper_golem.waxed_weathered_copper_button": "Waxed Weathered Copper Button",
-   "block.copper_golem.waxed_exposed_copper_button": "Waxed Exposed Copper Button",
-   "block.copper_golem.waxed_copper_button": "Waxed Copper Button",
+  "block.copper_golem.waxed_oxidized_copper_button": "Waxed Oxidized Copper Button",
+  "block.copper_golem.waxed_weathered_copper_button": "Waxed Weathered Copper Button",
+  "block.copper_golem.waxed_exposed_copper_button": "Waxed Exposed Copper Button",
+  "block.copper_golem.waxed_copper_button": "Waxed Copper Button",
 
-   "entity.copper_golem.copper_golem": "Copper Golem",
+  "entity.copper_golem.copper_golem": "Copper Golem",
 
-   "subtitles.entity.copper_golem.step": "Footsteps",
-   "subtitles.entity.copper_golem.ambient": "Copper Golem grunts",
-   "subtitles.entity.copper_golem.hurt": "Copper Golem hurts",
-   "subtitles.entity.copper_golem.no": "Copper Golem harrumphs",
-   "subtitles.entity.copper_golem.death": "Copper Golem dies",
-   "subtitles.entity.copper_golem.whirl": "Gear whirls"
+  "subtitles.entity.copper_golem.step": "Footsteps",
+  "subtitles.entity.copper_golem.ambient": "Copper Golem grunts",
+  "subtitles.entity.copper_golem.hurt": "Copper Golem hurts",
+  "subtitles.entity.copper_golem.no": "Copper Golem harrumphs",
+  "subtitles.entity.copper_golem.death": "Copper Golem dies",
+  "subtitles.entity.copper_golem.whirl": "Gear whirls"
 }
```

Comment:
Json files should preferably keep their terminating newline. Some might be missing them, so should be updated add them.